backbone/src/main/java/org/researchstack/backbone/model/survey/factory/SurveyFactory.java

```diff
@@ -168,7 +175,7 @@ public Step createSurveyStep(Context context, SurveyItem item, boolean isSubtask
                 if (!(item instanceof InstructionSurveyItem)) {
                     throw new IllegalStateException("Error in json parsing, ACCOUNT_EMAIL_VERIFICATION types must be InstructionSurveyItem");
                 }
-                return createEmailVerificationStep((InstructionSurveyItem)item);
+                return createEmailVerificationStep(context, (InstructionSurveyItem)item);
```

Comment:
Good point.  This can be any context, just used for string resources.  I think I mention that in the header of constructor of SurveyFactory.